src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -568,6 +569,7 @@ def parse_error_handler(parse_error):
 
   def _pex_run(self, pex, workunit_name, args, env):
     with self.context.new_workunit(name=workunit_name,
+                                   cmd=pex.cmdline(args),
```

Comment:
Does this mean we'll be computing the args twice? Once for the workunit, and another time to invoke?

Almost certainly not an issue for performance, but potentially an issue if they go out of sync for some reason.